my-app/src/components/register/register.js

```diff
@@ -43,65 +43,88 @@ addUser = _ => {
   alert("User " + user.username + " was registered");
 }
 
+addUserServer = _ => {
+  //Method that adds users to the database using the backend server.
+  const firstName = this.state.user.firstname;
```

Comment:
thanks for adding the backend functions into the register and login pages

Reply:
No worries